plugins/nodes/src/js/components/LeaderGrid.js

```diff
@@ -31,7 +31,7 @@ const ConfigurationRow = ({ keyValue, title, value }) => {
   );
 };
 
-export default function LeaderGrid({ master }) {
+export default function LeaderGrid({ leader }) {
```

Comment:
Should we rename this to MasterGrid then?

Reply:
I thought this one was addressed. I think the name LeaderGrid and NonLeaderGrid makes more sense as a pair as MasterGrid and NonLeaderGrid.